servisor.js

```diff
@@ -57,17 +58,26 @@ Servisor.prototype.run = function run (conf) {
     });
 };
 
-Servisor.prototype._sanitizeConfig = function (conf) {
+Servisor.prototype._sanitizeConfig = function (conf, options) {
     // TODO: Perform proper validation!
     if (!conf.logging) { conf.logging = {}; }
     if (!conf.metrics) { conf.metrics = {}; }
+    // check the number of workers to run
+    if(options.num_workers !== -1) {
+        // the number of workers has been supplied
+        // on the command line, so honour that
+        conf.num_workers = options.num_workers;
+    } else if(conf.num_workers === 'ncpu') {
```

Comment:
Should this just be a plain else, so that we default to ncpu if not otherwise set?